cypress/e2e/expenses/Delete.cy.js

```diff
@@ -0,0 +1,90 @@
+import '@testing-library/cypress/add-commands';
+
+import {
+  getUKFormattedEuros,
+  getUKFormattedDate,
+} from '../../../src/utils/helpers';
+
+describe('DELETE EXPENSE - NOT CURRENT MONTH', () => {
+  before(() => {
+    cy.intercept('GET', '**/items*', { fixture: 'expense-array' }).as(
```

Comment:
Sure it would have been much easier 😄 
I wanted to make this test more challenging for myself and simulate a slightly different real user action. 

Plus, one of the problems I had with the old intercepts was that I could not figure exactly how to make sure that we have a persistent expense state through the first GET we intercept.

Reply:
Ah gotcha, but wouldn't the test need to re-visit the page (refresh) in order to test if the expenses are really persisted? At any rate, testing state persistence might make a nice test unto itself.

Reply:
With persist I meant that the expenses were there but I could not do anything with them... I was having crazy problems at the beginning. 😄 

I think the **component testing functionality** might be a next cool step to learn cypress and check state persistence